src/txn/txn.c

```diff
@@ -2811,14 +2820,24 @@ __wt_verbose_dump_txn_one(
         break;
     }
 
+    WT_ERR(__wt_scr_alloc(session, 2048, &snapshot_buf));
+    WT_ERR(__wt_buf_fmt(session, snapshot_buf, "%s", "["));
+    for (i = 0; i < txn->snapshot_data.snapshot_count; i++)
+        WT_ERR(__wt_buf_catfmt(
+          session, snapshot_buf, "%s%lu", i == 0 ? "" : ", ", txn->snapshot_data.snapshot[i]));
```

Comment:
%llu here.

Reply:
@y123456yz

Reply:
hi, @luke-pearson 
First of all, thank you for your reply in PR(https://github.com/wiredtiger/wiredtiger/pull/10269). Because I am a linux technology stack and not familiar with windows development, I did not compile code in windows environment every time I submitted PR, causing similar problems to occur many times.

Today, I will get familiar with windows compilation of wiredtiger code, and I will inform you to review the code after confirming that the compilation of linux and windows system is OK.

And happy holidays to you.

thanks.

Reply:
/root/yyz/wiredtiger-yyz-master/wiredtiger/src/txn/txn.c:2827:34: error: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 5 has type ‘uint64_t’ {aka ‘long unsigned int’} [-Werror=format=]
           session, snapshot_buf, "%s%llu", i == 0 ? "" : ", ", txn->snapshot_data.snapshot[i]));

I have change it using PRIu64, This should be fine.
thanks.